### PR TITLE
Merge 1.7 into 2.0

### PR DESCRIPTION
**Pull request by LaurentPetard**

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/spec/remover/JobInstanceExportRemoverSpec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* global describe, it, expect */
```
Comment (tamarasaurus) [inline src/Pim/Bundle/EnrichBundle/Resources/public/spec/remover/JobInstanceExportRemoverSpec.js]: we don't have js specs anymore in 2.0, you can remove this and the others

Comment (LaurentPetard) [reply src/Pim/Bundle/EnrichBundle/Resources/public/spec/remover/JobInstanceExportRemoverSpec.js]: Removed

File: src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/job_instance/xlsx_product_import_show.yml [hunk 2]
```diff
@@ -88,6 +99,17 @@ extensions:
         config:
             title: pim_enrich.form.job_instance.subsection.last_executions
 
+    pim-job-instance-xlsx-product-import-show-properties-categories-column:
+        module: pim/job/common/edit/field/text
+        parent: pim-job-instance-xlsx-product-import-show-properties
+        position: 165
+        targetZone: global-settings
```
Comment (SamirBoulil) [inline src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/job_instance/xlsx_product_import_show.yml]: Did you find a fix for this ?

Comment (LaurentPetard) [reply src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/job_instance/xlsx_product_import_show.yml]: Yes but I forgot to fix this file...

**Pull request merged**